lib/cli.js

```diff
@@ -27,7 +27,7 @@ module.exports = function broccoliCLI(args) {
     .action(options => {
       actionPerformed = true;
 
-      const builder = getBuilder(options.brocfilePath);
+      const builder = getBuilder({ brocfilePath: options.brocfilePath });
```

Comment:
I'm unsure why we need to make a second object just to pass the single option into it. Any reason not to do either `getBuilder(options)` or `getBuilder(options.brocfilePath)` here?

Reply:
was planning to pass more options here, like `cwd`. But have to see how other options will be passed around to builder/watcher. As `build` and `serve` are going to share options, probably this is going to change.

Reply:
if you prefer I can simplify it now and change in upcoming PRs

Reply:
Nah, seems totally fine, I was mostly just curious.  Answer is: planning more changes that will leverage the object. 👍